x/launch/keeper/keeper_test.go

```diff
@@ -1,38 +1,50 @@
 package keeper
 
 import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/tendermint/spn/testutil/sample"
+	profilekeeper "github.com/tendermint/spn/x/profile/keeper"
```

Comment:
The import sort lint is only enabled into the Makefile because the lint can show the starport placeholders as an issue. But maybe we can put back this rule in the CI fixing these cases. We need to add a line after and before all placeholder comment's

Reply:
> We need to add a line after and before all placeholder comment's

That could be the way to go, I may think we don't have many placeholders in the `import` in the templates. This could be an improvement for the templates we could specify in https://github.com/tendermint/starport/issues/1409